samples/squidb-ios-sample/squidb-ios-sample/squidb-gen/com/yahoo/squidb/sample/models/Tag.java

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
No, this is in the sample project -- we check in the iOS generated files. I can't remember if there was a reason for that or if it happened accidentally; I can look into it to find out, but separate issue.